src/main/java/org/jenkinsci/extension_indexer/ExtensionPointListGenerator.java

```diff
@@ -304,7 +304,7 @@ private MavenRepositoryImpl createRepository() throws Exception {
      * Walks over the plugins, record {@link #modules} and call {@link #discover(Module)}.
      */
     private void processPlugins(MavenRepository r) throws Exception {
-        ExecutorService svc = Executors.newFixedThreadPool(4);
+        ExecutorService svc = Executors.newFixedThreadPool(1);
```

Comment:
Yes and no.

Yes, because things go slower (while they're still going).

No, because it kept locking up on me with stack traces like https://gist.github.com/daniel-beck/401a85db7d58a55b0c78e02d33f971fd